sdk/spring/pipeline/cosmos-integration-matrix.json

```diff
@@ -10,6 +13,9 @@
       }
     },
     "TestFromSource": true,
-    "JavaTestVersion": ["1.11", "1.17", "1.21"]
+    "JavaTestVersion": ["1.11", "1.17", "1.21"],
```

Comment:
only cosmos support 1.11, should we align this with ours?

Reply:
Let me remove it.

Reply:
Removed via PR https://github.com/Azure/azure-sdk-for-java/pull/41227